core/services/workflows/events/types.go

```diff
@@ -28,6 +28,18 @@ const (
 	// SchemaUserLogs represents the schema for user log events
 	SchemaUserLogs string = "/cre-events-user-logs/v1"
 
+	// V2 event constants
+	WorkflowExecutionStartedV2    string = "WorkflowExecutionStartedV2"
```

Comment:
@patrickhuie19 
We should use `WorkflowExecutionStarted` here as thats the message name of the proto here
https://github.com/smartcontractkit/chainlink-protos/blob/main/workflows/workflows/v2/workflow_execution_started.proto#L10

The version is already present in the proto package name `workflows.v2` so there is no need to add a version postfix to the message name

```suggestion
	WorkflowExecutionStartedV2    string = "WorkflowExecutionStarted"
```

Reply:
We can't do that because it will conflict with the `WorkflowExecutionStarted` const that lives right above in this file on L8

Reply:
Protos conflict when the full name is the same, in your case the full name will be different 
`workflows.v2.WorkflowExecutionStarted` (new) vs `workflows.v1.WorkflowExecutionStarted` (old)